mysql-test/suite/rpl/t/rpl_update_partitioned_table_with_compressed_column.test

```diff
@@ -0,0 +1,20 @@
+#
+# PS-7599 : Node inconsistency seen in PS during update on compressed column table
+#
+
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
+
+DROP TABLE IF EXISTS tt_2_p;
+
+CREATE TABLE tt_2_p (f1 INT, f2 BLOB COLUMN_FORMAT COMPRESSED) ENGINE=INNODB PARTITION BY HASH (f1) PARTITIONS 2;
+
+REPLACE INTO tt_2_p(f1,f2)VALUES(0,'123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456');
+
+UPDATE tt_2_p SET f1=1 WHERE f1 = 0;
+
+--source include/sync_slave_sql_with_master.inc
+
+DROP TABLE tt_2_p;
```

Comment:
Including`include/sync_slave_sql_with_master.inc` changes the connection to slave in the end.
So, this DELETE is executed on slave and not on the master.

So, please use
```
--source include/rpl_connection.inc
DROP TABLE tt_2_p;
--source include/rpl_end.inc
```